README.md

```diff
@@ -56,3 +56,7 @@ Use a [simple webpack config](https://github.com/facebook/create-react-app/issue
 - `yarn countries`
 
 Generate the JSON file `src/assets/countries.json` which contains all the countries available on OFF taxonomy. Data are obtained from static.openfoodfacts.org
+
+## How you can help
+- https://wiki.openfoodfacts.org/Hunger_Games
+- [Help translate Hunger Games to your language](https://translate.openfoodfacts.org/translate/openfoodfacts/1942/en-fr?filter=basic&value=0)
```

Comment:
With the link to the wiki, I don't know what people can do there/ Probably better to redirect them to the slack channel

```suggestion
## How you can help

For developers, you can have a look at [good first issues](https://github.com/openfoodfacts/hunger-games/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) or ask on [slack](https://slack.openfoodfacts.org/) in hunger-games channel.

You can also help by improving [translation in your language](https://translate.openfoodfacts.org/translate/openfoodfacts/1942/en-fr?filter=basic&value=0), Or the [wiki page](https://wiki.openfoodfacts.org/Hunger_Games) of the project
```